core/src/main/java/bisq/core/locale/GlobalSettings.java

```diff
@@ -47,6 +48,10 @@ public static void setUseAnimations(boolean useAnimations) {
         GlobalSettings.useAnimations = useAnimations;
     }
 
+    public static void setUseDarkMode(boolean useDarkMode) {
+        GlobalSettings.useDarkMode = useDarkMode;
```

Comment:
I don't think we need to store it in the GlobalSettings

Reply:
Again, this is based on an earlier version of #2414.  As per your feedback, the value is not stored in `GlobalSettings` anymore.

Reply:
Okay, I'll remove the GlobalSettings. FYI @peterzen this code is not based on #2414, I merely copied the setUseAnimations directly above it and changed the name to setUseDarkMode.